awx/api/serializers.py

```diff
@@ -4167,6 +4167,9 @@ class NotificationTemplateSerializer(BaseSerializer):
     class Meta:
         model = NotificationTemplate
         fields = ('*', 'organization', 'notification_type', 'notification_configuration', 'messages')
+        # preserve metadata for `messages` so default value can be shown for users with read-only access
+        preserve_metadata = {'messages': 'default'}
```

Comment:
it's more like `preserve_get_metadata`, the use of this is very narrow

Reply:
true, updating 👍